content/20.treatments.md

```diff
@@ -11,48 +11,96 @@ In the following sections, we critically appraise the literature surrounding the
--Given what we know about the mechanisms of the virus and why it produces the symptoms we see, are there drugs or categories of intervention that might be relevant?
-
-So far, some strategies for reducing the severity of symptoms have included...
-
-#### Symptom Management Approach 1
-
-Brief background on the therapeutic.
+The clinical picture of SARS-CoV-2 infection differs dramatically between individuals.
+Some are asymptomatic.
+Others experience mild COVID-19 symptoms, such as cough, sore throat and fever, while the most severe cases of COVID-19 include severe complications including pneumonia and Acute Respiratory Distress Syndrome (ARDS), which can lead to respiratory failure and death [@url:https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html]. 
+Vaccines are one avenue to mitigate harm from viral pathogens, but in the case of a rapidly growing pandemic the longer timeframe of vaccine development and distribution means that there can be a key for treatments that palliate symptoms to avoid the most severe outcomes from infection. 
+
+#### Tocilizumab
+
+A recent study carried out on a sample of 191 adult COVID-19 in-patients at two Wuhan hospitals found that blood samples taken at admission contained significantly higher concentrations of interleukin-6 (IL-6) in patients who ultimately deceased compared to those who survived; average concentrations of IL-6 remained higher in the deceased group than the surviving group throughout hospitalization [@doi:ggnxb3].
+This suggests that these individuals may be experiencing a "cytokine storm", which refers to an excessive inflammatory response.
+IL-6 plays a key role in this response [@doi:10.1128/mmbr.05015-11].
+IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, which are immune system regulators that are primarily responsible for immune cell differentiation. 
+Specifically, IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells [@pmid:9052836] and acts as a growth factor for hybridoma and myeloma cells [@pmid:8431556; @pmid:2104241]. 
+In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [10.1158/2326-6066.CIR-14-0022].
+In this way interleukins regulate both the pro- and anti-inflammatory responses.
+In this context, the observation of elevated IL-6 in patients who died may reflect an over-production of proinflammatory interleukins. 
+
+In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
+It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽 [doi.org/10.1016/j.virusres.2007.02.007, DOI: 10.1002/path.2067].
```

Comment:
```suggestion
The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α [@doi:10.1016/j.virusres.2007.02.007], and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1𝛽, and TNF-α expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy [@doi:10.1002/path.2067].
However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
```

Reply:
Leaving in the BALF result for now because it might be interesting to dig further into this topic!